dotcom-rendering/src/client/sentryLoader/sentry.ts

```diff
@@ -55,10 +55,21 @@ if (
 	Sentry.setTag('dcr.bundle', dcrJavascriptBundle('Variant'));
 }
 
-export const reportError = (error: Error, feature?: string): void => {
+export const reportError = (
+	error: Error,
+	tags?:
+		| string
+		| {
+				[key: string]: string;
+		  },
+): void => {
```

Comment:
I wonder if an improvement would be to make feature mandatory and separate it from an optional tags parameter rather than overloading a single parameter?

We don't have to maintain backwards compatibility given we can update DCR and we'll be migrating commercial to use this method.

```suggestion
export const reportError = (
	error: Error,
	feature: string,
	tags?:
	    {
				[key: string]: string;
		  },
): void => {
```

Reply:
Further to Ravi's point, if we don't have to maintain backwards compatibility, do we need the `feature` param at all? Isn't `feature` just a key of a tag?

Reply:
Thanks both 🙏

I agree `feature` should continue to be a mandatory argument. It's more approachable to do this using an explicit argument, rather than using a special `CustomTags` type. 

Of course one day someone will provide `feature` both as an argument and also as a property on the `tags` object, but watchagonnado?

This is a safer implementation path too 👍

```typescript
reportError(
  error: Error, 
  feature: string, 
  tags?: { [key: string]: string; }
): void
```